### ISSUE TITLE
Most people will get dfu loaded Tomu boards

### ISSUE THREAD

**Issue by mithro** [im-tomu/im-tomu.github.io#18]
Would probably be a good idea to focus on the dfu loading rather than the old serial bootloader. Most people will be getting their boards from crowdsupply which comes preloaded...

**Comment by ringerc** [comment]
Agreed. Not only that, I found the instructions for the serial bootloader to be insufficient anyway - see http://blog.ringerc.id.au/2019/01/updating-tomu-bootloader-on-tomuim.html . They don't link to the firmware to use and I found that `screen` didn't work for me, I had to use `minicom` then figure out the archaic UI for xmodem uploads.

I suggest moving the legacy bootloader info to another page. On the main page just mention that you can tell if you have a tomu with an old firmware that needs updating (link) if:

* no lights blink when you plug it in
* `dmesg` shows something like

        [14275.148309] usb 1-2: New USB device found, idVendor=10c4, idProduct=0003, bcdDevice= 0.00
        [14275.148321] usb 1-2: New USB device strings: Mfr=1, Product=2, SerialNumber=0
        [14275.148329] usb 1-2: Product: EFM32 USB CDC serial port device
        [14275.148336] usb 1-2: Manufacturer: Silicon Laboratories Inc.
        [14275.155055] cdc_acm 1-2:1.0: ttyACM0: USB ACM device

    when it's connected;
* `lsusb` shows `ID 10c4:0003`; and
* it doesn't show up in `sudo dfu-util -l`

wheras on Tomuboot, assuming it's not booting straight into a program, you'll see:

* `sudo dfu-util -l` will list it as e.g.

         Found DFU: [1209:70b1] ver=0101, devnum=55, cfg=1, intf=0, path="1-2", alt=0, name="Tomu Bootloader (5) v2.0-rc7", serial="UNKNOWN";

* `dmesg` will report output like:

         [19998.971118] usb 1-2: new full-speed USB device number 56 using xhci_hcd
         [19999.099659] usb 1-2: New USB device found, idVendor=1209, idProduct=70b1, bcdDevice= 1.01
         [19999.099669] usb 1-2: New USB device strings: Mfr=1, Product=2, SerialNumber=0
         [19999.099675] usb 1-2: Product: Tomu Bootloader (5) v2.0-rc7
         [19999.099681] usb 1-2: Manufacturer: Kosagi

    ; and
* `lsusb` will show a device `ID 1209:70b1`

**Comment by ringerc** [comment]
The legacy firmware update page is free to re-use any of the material on the above blog, too.
